xml/gl.xml

```diff
@@ -79,7 +79,14 @@ typedef unsigned int GLhandleARB;
         <type>typedef void (<apientry/> *<name>GLVULKANPROCNV</name>)(void);</type>
     </types>
 
-    <!-- SECTION: GL parameter class type definitions. -->
+    <!-- SECTION: GL parameter class type definitions.
+         The groups are intended to contain all the possible legal values
+         for corresponding function parameters, but it is likely that many
+         of the groups are out of date relative to current OpenGL and OpenGL
+         ES specifications, and the many extensions to those specifications.
```

Comment:
We should make the reliability (or lack thereof) of the groupings more explicit.
```suggestion
         ES specifications, and the many extensions to those specifications.
         As such, they may not be a reliable source for enumeration info.
```
or
```suggestion
         ES specifications, and the many extensions to those specifications.
         You should prefer the GL enumerant definitions where possible.
```

Reply:
Jon applied the first suggestion without marking the review as resolved ;) As for suggestion 2, while moot, I was referring to the `<enums namespace=` portion of the spec.